docs/team/chen-yiyang.md

```diff
@@ -5,42 +5,41 @@ title: Chen Yiyang's Project Portfolio Page
 
 Given below are my contributions to the project.
 
-* **New Feature**: Added the ability to undo/redo previous commands.
-    * What it does: allows the user to undo all previous commands one at a time. Preceding undo commands can be reversed by using the redo command.
-    * Justification: This feature improves the product significantly because a user can make mistakes in commands and the app should provide a convenient way to rectify them.
-    * Highlights: This enhancement affects existing commands and commands to be added in future. It required an in-depth analysis of design alternatives. The implementation too was challenging as it required changes to existing commands.
-    * Credits: *{mention here if you reused any code/ideas from elsewhere or if a third-party library is heavily used in the feature so that a reader can make a more accurate judgement of how much effort went into the feature}*
+- **New Feature**: Added the ability to mark visits as done.
+  - What it does: allows the user to make the next visit to an elderly as done using elderly's index. When a visit is marked as done, the `LastVisit` field will be updated. If the visit marked is recurring, it will be replaced with the next one following the recursion frequency.
+  - Justification: This feature improves the product significantly because a user can proceed with a sequence of visits of an elderly and visits are at the core of SeniorLove.
+  - Highlights: This command is dependent on the `Visit`, `Frequency`, `Occurrence` and `LastVisit` fields of an elderly in the system. Implementation for the command was challenging as many scenarios need to be taken into account. Defensive programming was also used to ensure smooth running of the programme should certain fields involved be missing or invalid.
 
-* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
 
-* **Code contributed**: [RepoSense link]()
+- **New Feature**: Added the ability to view incoming visits.
+  - What it does: allows the user to see all the visits that are scheduled in the next 7 or 30 days depending on the command parameter.
+  - Justification: This feature improves the user experience as the user can view incoming visits easily and plan their schedule accordingly.
+
+- **New Features**: Implemented Datetime for all time-related fields
+  - What it does: Replaces the original Date implementation with Datetime for all time-related fields such as `Visit` and `LastVisit`.
+  - Justification: This feature enhances the project by making information more accurate and relevant given that visits for the elderly are of high importance in the real world.
```

Comment:
Not too sure if this classifies as a new feature or an enhancement

Reply:
Okay I will move it to enhancement.